dotnet-test-samples/kinesis-lambda-dynamodb/tests/KinesisEventHandler.UnitTests/Functions/ProcessEmployeeFunctionTests.cs

```diff
@@ -0,0 +1,80 @@
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Amazon.Lambda.TestUtilities;
+using FluentAssertions;
+using KinesisEventHandler.Functions;
+using KinesisEventHandler.Repositories;
+using KinesisEventHandler.Repositories.Mappers;
+using KinesisEventHandler.Repositories.Models;
+using KinesisEventHandler.UnitTests.Utilities;
+using Moq;
+using Xunit;
+
+namespace KinesisEventHandler.UnitTests.Functions;
+
+public class ProcessEmployeeFunctionTests
+{
+    [Fact]
+    public Task ProcessEmployeeFunction_Should_ExecuteSuccessfully()
```

Comment:
What is the end result, beside not throwing an exception

Reply:
Just dumping the record to DynamoDB